### PR TITLE
Homepage, index and show views for products, seeds.rb changes, schema changes

### PR DESCRIPTION
**Pull request by njch5**
Created/styled homepage, index, and show views for products (along with nested categories and merchants). Also fixed the seeds.rb file so everyone should run `rails db:reset` to get the latest seeds.

Changes to the schema: added default photo to photo_url and created merchant_id to order_items.

### CODE REVIEW

File: app/controllers/products_controller.rb [hunk 1]
```diff
@@ -1,62 +1,76 @@
...
+      category = Category.find_by(id: params[:category_id])
+      if category == nil
+        return head :not_found
+      else
+        @products = category.products
+      end
+    else
+      @products = Product.where(status: true)
+    end
+    # @products = Product.where(status: true)
```
Comment (rinostar) [inline app/controllers/products_controller.rb:23]: My branch PR will have some merge conflict with you here, but it should be very similar, so plz feel free to take a look and keep the version you prefer :)

File: app/controllers/products_controller.rb [hunk 2]
```diff
@@ -1,62 +1,76 @@
...
   end
 
   def new
     @product = Product.new
   end
```
Comment (rinostar) [inline app/controllers/products_controller.rb:28]: I updated this action to take into consideration of the nested routes for merchant and product.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -1,2 +1,15 @@
...
+    @current_merchant ||= Merchant.find(session[:user_id]) if session[:user_id]
+  end
+
+  private
+
+  def find_merchant
+    if session[:user_id]
+      @login_merchant = Merchant.find_by(id: session[:user_id])
+    end
+  end
```
Comment (YasminM11) [inline app/controllers/application_controller.rb:14]: looks good!

**Review by rinostar — APPROVED** [review]
Thank you Nicky! Great job <3

**Review by YasminM11 — APPROVED** [review]
Thank you

**Pull request merged**